keepa/_version.py

```diff
@@ -1,6 +1,6 @@
 """Version number for keepa
 """
 # major, minor, patch, -extra
-version_info = 1, 1, 0
+version_info = 1, 2, 0
```

Comment:
It's about time to have a minor update, so I'll tag this release and upload to PyPi.